run_pacta_data_preparation.R

```diff
@@ -890,6 +891,38 @@ for (pkg_name in pacta_packages) {
   )
 }
 
+# Create archive files
+if (export_archives) {
+  logger::log_info("Exporting input and output archives.")
+
+  logger::log_debug("Creating inputs zip file.")
+  inputs_zip_file_path <- paste0(data_prep_outputs_path, "_inputs.zip")
+  logger::log_trace("Zip file path: \"{inputs_zip_file_path}\".")
+  zip(
+    zipfile = inputs_zip_file_path,
+    files = normalizePath(unlist(parameters[["input_filepaths"]])),
+    extras = c(
+      "--junk-paths", # do not preserve paths
+      "--no-dir-entries", # do not include directory entries
+      "--quiet" # do not print progress to stdout
+    )
+  )
+  logger::log_debug("Inputs archive created.")
+
+  logger::log_debug("Creating outputs zip file.")
+  outputs_zip_file_path <- paste0(data_prep_outputs_path, ".zip")
```

Comment:
Is this dangerous? Effectively, this will drop a zip file in the parent directory of the specified output directory, which could be inaccessible. Same with the `inputs_zip_file_path`

Reply:
this is less of a problem with latest changes to #153 to make the timestamped output directory non-optional?

Reply:
In general, this is intended to be used in conjunction with the changes in #153. I split it out into a separate PR on your request. 

But in general, if someone if using a generic `./outputs` and `./inputs` (not timestamped), then exporting the archive is probably a bad idea. But since I've made the default in #153 to use time unique directories, that dangerous behavior is explicitly opt-in.

Reply:
The only time I would expect this to path to be unwritable is when the user is writing directly to the directory which is volume mounted to the docker container. That's pretty far off-label usage so I'm not particularly worried about it, but if you want to flag an issue to do a warning check, I can add it in later.

Reply:
with #153 merged and it forcing outputs into a subdirectory, and therefore the zip landing in the originally specified `data_prep_outputs_path`, I think this is not a concern anymore